server/controllers/logstreaming_controller.go

```diff
@@ -111,14 +114,17 @@ func (j *JobsController) GetProjectJobs(w http.ResponseWriter, r *http.Request)
 }
 
 func (j *JobsController) GetProjectJobsWS(w http.ResponseWriter, r *http.Request) {
+	jobsMetric := j.StatsScope.Scope("get_project_jobs_ws")
```

Comment:
Our metrics should be classified into top level scopes. Right now we have `atlantis.cmd...` as one top level. This should fit under another `atlantis.api`

I would suggest separating the scope for the api and the websocket.

so you'd have:

```
api.getprojectscope
             .success
             .error
             .latency
             .websocket <- this scope would be used for websocket connections as well.
```